client/assets/stylesheets/shared/_typography.scss

```diff
@@ -1,18 +1,9 @@
-// Brand font
-
-@import '~@automattic/typography/sass/fonts';
+@import '~@automattic/typography/sass/font-brand';
```

Comment:
The `font-brand` module doesn't contain just variable or mixin definitions, but actual CSS code that's written to the output. That causes a lot of duplication.

Can we move the `font-brand` import to _assets/stylesheets/style.scss_ and include only the `font-family` and `font-size` modules here? This module is part of the SASS loader prelude and is present in every `scss` file compilation.

_assets/stylesheets/style.scss_ is the root stylesheet of the Calypso app, and it contains other similar global styles: CSS variables for the color schemes, definitions for named animations... Looks like a perfect place for the Recoleta `@font-face` and the `.wp-brand-font` rule.